src/pages/Home.js

```diff
@@ -0,0 +1,40 @@
+import {Link} from 'react-router-dom';
+import styled from 'styled-components';
+
+export default function Home() {
+  return (
+    <>
+      <Heading>Find your perfect chair</Heading>
+      <MainContainer>
+        <StyledLink to={`./industrial`}>Industrial</StyledLink>
+        <StyledLink to={`./scandinavian`}>Scandinavian</StyledLink>
+        <StyledLink to={`./contemporary`}>Contemporary</StyledLink>
+        <StyledLink to={`./mid-century-modern`}>Mid-Century Modern</StyledLink>
```

Comment:
The path under the attribute `to` should be equal to the `path` of your route (in App.js):

```suggestion
        <StyledLink to={`/industrial`}>Industrial</StyledLink>
        <StyledLink to={`/scandinavian`}>Scandinavian</StyledLink>
        <StyledLink to={`/contemporary`}>Contemporary</StyledLink>
        <StyledLink to={`/mid-century-modern`}>Mid-Century Modern</StyledLink>
```